build/Common.prod.props

```diff
@@ -2,13 +2,6 @@
   <Import Project=".\Common.props" />
 
   <PropertyGroup>
-    <!-- OmniSharp/VS Code requires TargetFrameworks to be in descending order for IntelliSense and analysis. -->
```

Comment:
FYI - I moved all the TFM info to `Common.props` instead of having separation of prod/nonprod, I think having all TFM info in a single place would make it much easier to introduce new runtime version / deprecate old runtime / ensure consistency.